opal-gwt-client/src/main/java/org/obiba/opal/web/gwt/app/client/project/genotypes/ProjectGenotypesPresenter.java

```diff
@@ -175,21 +176,23 @@ public void initialize(ProjectDto dto) {
   }
 
   private void initializeParticipantIdentifierMappingList() {
-    String uri = UriBuilders.IDENTIFIERS_TABLE_VARIABLES.create().build("keys");
-    ResourceRequestBuilderFactory.<JsArray<VariableDto>>newBuilder() //
-            .forResource(uri) //
-            .withCallback(new ResourceCallback<JsArray<VariableDto>>() {
-              @Override
-              public void onResource(Response response, JsArray<VariableDto> resource) {
-                participantIdentifiersMappingList.clear();
-                JsArray<VariableDto> variableDtoJsArray = JsArrays.toSafeArray(resource);
-                Iterable<VariableDto> variableDtos = JsArrays.toIterable(variableDtoJsArray);
-                for (VariableDto variableDto : variableDtos) {
-                  participantIdentifiersMappingList.add(variableDto.getName());
-                }
-              }
-            }) //
-            .get().send();
+    Map<String, String> params = Maps.newHashMap();
+    params.put(MAPPINGS_ENTITY_TYPE_PARAM, PARTICIPANT_ENTITY_TYPE);
+
+    String uri = UriBuilders.IDENTIFIERS_MAPPINGS.create().query(params).build();
+    ResourceRequestBuilderFactory.<JsArray<IdentifiersMappingDto>>newBuilder()
+        .forResource(uri)
+        .withCallback(new ResourceCallback<JsArray<IdentifiersMappingDto>>() {
+          @Override
+          public void onResource(Response response, JsArray<IdentifiersMappingDto> resource) {
+            JsArray<IdentifiersMappingDto> identifiersMappings = JsArrays.toSafeArray(resource);
+            participantIdentifiersMappingList.clear();
+            for(IdentifiersMappingDto identifiersMappingDto : JsArrays.toIterable(identifiersMappings)) {
+              participantIdentifiersMappingList.add(identifiersMappingDto.getName());
```

Comment:
You need to filter the mappings for entity type *Participant* only. See *entityTypes* property of *IdentifiersMappingDto*.